packages/amazonq/test/unit/amazonq/lsp/lspClient.test.ts

```diff
@@ -0,0 +1,37 @@
+import * as sinon from 'sinon'
+import assert from 'assert'
+import { LspClient } from 'aws-core-vscode/amazonq'
+
+describe('Test LSP client', function () {
```

Comment:
It is never necessarily to mention "Test" in a test name. Nor "should", "correct", etc. 
```suggestion
describe('Amazon Q LSP', function () {
```